### DISCUSSION TITLE
SPI speed

### DISCUSSION THREAD

**Discussion by parabyte** [witnessmenow/ESP32-Cheap-Yellow-Display#236] [General]
im using the spi interface pins for spi ethernet device using espressif's ETH.h library for sanity reasons has anyone here had issues setting spi speed to anything higher then 20mhz?

**Comment by clach04** [comment]
https://github.com/peterhinch/ has some notes on this. The specs say 10 BUT it seems like most people are able to use 40 without any issues.

I actually need to spend some time experimenting with different speeds as I'm [using 10](https://github.com/clach04/cyd_clocks/blob/main/color_setup.py#L54..L56) with [nano-gui](https://github.com/peterhinch/micropython-nano-gui) and getting more [audio interference](https://github.com/clach04/cyd_clocks/issues/48) compared with the raw drivers https://github.com/rdagger/micropython-ili9341 which I'm [running at 40](https://github.com/clach04/cyd_clocks/blob/a684fa9ae211df1f238cbfa9b16dc60778bdc980/cyd_wrap.py#L57..L58).  

@parabyte what did you end up doing? What are your observations?

**Comment by clach04** [comment]
so 40Mhz is a negative in my scenario, the interference beep is significantly louder with 40Mhz.
